source/manuals/programming-languages/css.html.md.erb

```diff
@@ -9,144 +9,71 @@ review_in: 6 months
 ## Contents
 
 * [Whitespace](#whitespace)
-* [Dimensions](#dimensions)
-* [Bleeding edge features](#bleeding-edge-features)
+* [Spacing](#spacing)
+* [Vendor prefixing](#vendor-prefixing)
 * [Sass nesting](#sass-nesting)
 
 ## Whitespace
 
 Use soft tabs with a 2 space indent. This follows the conventions we use in our other projects.
 
-## Dimensions
+## Spacing
 
-Dimensions for layout should either use the [variables from the
-  frontend_toolkit][measurements.scss] or be in pixel values in multiples of `5px`.
+You should use the [GOV.UK Design System spacing scale](https://design-system.service.gov.uk/styles/spacing/).
 
-  [measurements.scss]: https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_measurements.scss
-
-  ```css
-  // Bad
-  .block {
-    margin: 28px;
-  }
-  .block {
-    padding: 12px 19px;
-  }
+### Use with deprecated libraries
 
-  // Good
-  .block {
-    margin: $gutter;
-  }
-  .block {
-    padding: 10px 20px;
-  }
-  ```
+If your project is not using the GOV.UK Design System, you should use the [variables from the
+  frontend_toolkit][measurements.scss] or use pixel values in multiples of `5px`.
```

Comment:
Some stray spaces at the beginning of this line, bit thrown by that!

Reply:
Oh that is weird, I'll try and sort them.